odbc/src/api/cancel_tests.rs

```diff
@@ -0,0 +1,142 @@
+use crate::{handles::definitions::*, SQLCancel};
+use definitions::SqlReturn;
+use mongo_odbc_core::MongoConnection;
+use mongodb::sync::Client;
+use std::env;
+
+mod integration {
+
+    use super::*;
+
+    fn generate_connection_uri() -> String {
+        let user_name = env::var("ADF_TEST_LOCAL_USER").expect("ADF_TEST_LOCAL_USER is not set");
+        let password = env::var("ADF_TEST_LOCAL_PWD").expect("ADF_TEST_LOCAL_PWD is not set");
+        let host = env::var("ADF_TEST_LOCAL_HOST").expect("ADF_TEST_LOCAL_HOST is not set");
+        format!("mongodb://{user_name}:{password}@{host}")
+    }
+
+    // skip-reason: SQL-1959
+    // the local adf currently can't be configured to handle killop commands; it is on be default in production
```

Comment:
```suggestion
    // the local adf currently can't be configured to handle killop commands; it is on by default in production
```
nit

Reply:
thank you for catching this, i fear I am quite prone to typos/misspelling in my comments 😔